src/library/dynamo.js

```diff
@@ -121,7 +121,7 @@ export const batchPutIntoDynamoDb = async (AWS, recs, tName,
     if (unprocessedRecords.length > 0) {
       await sleep(backoff);
       return batchPutIntoDynamoDb(AWS, unprocessedRecords, tName,
-        backoff + 1000);
+        backoff + 1000); // CR: Mickey: Why are we increasing the sleep time on each iteration?
```

Comment:
@MickeyMcc AWS actually wants us to increase this exponentially (https://docs.aws.amazon.com/general/latest/gr/api-retries.html). I am more optimistic and only increasing it linearly.